### ISSUE TITLE
Advanced buildmode mass fill doesn't use the object from middle-click copy

### ISSUE THREAD

**Issue by gbasood** [d3athrow/vgstation13#13313]
Looks like #11553 is happening, AGAIN
Thanks

**Comment by Intigracy** [comment]
But that fixed it didn't it

**Comment by gbasood** [comment]
It was fixed once
But if you try to mass-fill instead of single-clicking it will only spawn lockers again

**Comment by Intigracy** [comment]
What specifically are you doing that causes this issue?

**Comment by JustSumBody** [comment]
Yeah, it now seems to only work for everything BUT mobs as opposed to before when it only worked for mobs apparently
WHOOPS

**Comment by Exxion** [comment]
So I guess `copycat` just doesn't work with fill

**Comment by west3436** [comment]
Note that this applies only to mobs
